core/txflow/src/txflow_task/mod.rs

```diff
@@ -146,18 +152,23 @@ impl<'a, P: Payload, W: WitnessSelector> TxFlowTask<'a, P, W> {
     }
 
     /// Processes the incoming message.
-    fn process_incoming_message(&mut self, message: SignedMessageData<P>, message_sender: UID, requires_reply: bool) {
+    fn process_incoming_message(
+        &mut self,
+        message: SignedMessageData<P>,
+        message_sender: UID,
+        requires_reply: bool,
+    ) {
         // Check one of the optimistic scenarios when we already know this message.
         if self.dag_as_ref().contains_message(message.hash)
-            || self.blocked_messages.contains_key(&message.hash) {
+            || self.blocked_messages.contains_key(&message.hash)
+        {
             return;
         }
 
-        let unknown_hashes: HashSet<TxFlowHash> = (&message.body.parents).iter().filter_map(
-            |h| if self.dag_as_ref().contains_message(*h) {
-                None } else {
-                Some(*h)
-            } ).collect();
+        let unknown_hashes: HashSet<TxFlowHash> = (&message.body.parents)
+            .iter()
+            .filter_map(|h| if self.dag_as_ref().contains_message(*h) { None } else { Some(*h) })
```

Comment:
It is common in Rust to write ternary operator like it was written before. The formatter is not familiar with it.